src/LedgerArk.js

```diff
@@ -91,15 +91,15 @@ LedgerArk.prototype.signTransaction_async = function(path, rawTxHex) {
 }
 
 LedgerArk.prototype.getAppConfiguration_async = function() {
-	var buffer = new Buffer(5);
+	var buffer = Buffer.from(5);
```

Comment:
If you wish to create a new Buffer of a specific length you need to use `Buffer.alloc`